test/test_torch.py

```diff
@@ -4535,12 +4430,12 @@ def test_nonzero(self):
                     # This test will allow through some False positives. It only checks
                     # that the elements flagged positive are indeed non-zero.
                     for i in range(dst1.size(0)):
-                        self.assertNotEqual(tensor[dst1[i, 0], dst1[i, 1]], 0)
+                        self.assertNotEqual(tensor[dst1[i, 0], dst1[i, 1]].item(), 0)
```

Comment:
this needs a .item()?  (same comment as above)?

Reply:
Yeah, because the way we override assertNotEqual on Tensors to use assertGreatEqual, which calls `abs` which doesn't work on ByteTensors.